sdk/tables/perf-tests/data-tables/test/tables.spec.ts

```diff
@@ -10,6 +10,7 @@ export abstract class TablesTest<TOptions = Record<string, unknown>> extends Per
     super();
     const connectionString = getEnvVar("SAS_CONNECTION_STRING");
     this.client = TableClient.fromConnectionString(connectionString, tableName);
+    this.client.pipeline.addPolicy(testProxyHttpPolicy(this.getHttpClient()));
```

Comment:
Similar to my previous comment, can this be made more generic?  Is there a convention that core v2 clients will have a `pipeline` property?  If so, I recommend:

```js
this.client = configureClient(TableClient.fromConnectionString(connectionString, tableName));
```

Where the `PerfStressTest.configureClient()` method calls `pipeline.addPolicy()` on whatever client object is passed in. This way, we can extend the functionality by setting more/different policies without changing all test classes.

Reply:
this.configureClient(TableClient.fromConnectionString(connectionString, tableName));